### PR TITLE
Add `pug` extension for PugJS(Jade Template)

### PR DESCRIPTION
**Pull request by tw-statics**

### CODE REVIEW

**Comment by flosse** [comment]
would you mind to reduce your PR into one commit?

File: src/sloc.coffee [hunk 1]
```diff
@@ -272,6 +272,7 @@ extensions = [
   "iced"
   "ino"
   "jade"
+  "pug"
   "java"
   "jl"
   "js"
```
Comment (pfrocha) [inline src/sloc.coffee]: jsp

Comment (pfrocha) [inline src/sloc.coffee]: apl

Comment (pfrocha) [inline src/sloc.coffee]: app

Comment (pfrocha) [inline src/sloc.coffee]: jrxml

**Comment by flosse** [comment]
> Gostaria que colocasse estas extensões.
Tem jeito?

Would you mind to translate this into english? Thanks!

**Comment by FranzSkuffka** [comment]
I think there is a squash & rebase option in the merge dropdown :)

**Comment by flosse** [comment]
I'll close this for now. Feel free to reopen if needed :)

**Pull request closed without merging**